src/Sylius/Behat/Context/Api/Admin/ManagingLocalesContext.php

```
@@ -148,17 +129,13 @@ public function iShouldBeNotifiedThatTheCodeIsInvalid(): void
         Assert::same($this->responseChecker->getError($response), 'code: This value is not a valid locale code.');
     }
 
-    /**
-     * @Then I should be informed that locale :localeCode has been deleted
-     */
+    #[Then('I should be informed that locale :localeCode has been deleted')]
     public function iShouldBeInformedThatLocaleHasBeenDeleted(string $localeCode): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$localeCode'. (undefined)

(UnusedFormalParameter)